### DISCUSSION TITLE
Using the Popover component on Avatar in AvatarGroup

### DISCUSSION THREAD

**Discussion by Kayoshi-dev** [mantinedev/mantine#1338] [Q&A] [answered]
First, Hello and thank you for this awesome library.

I'm trying to use the Popover or Tooltip component on an Avatar in an Avatar Group but without success. (But it works on Avatar component when it's not in an AvatarGroup component).

Basically what I'm trying do to now is :

```jsx
<AvatarGroup>
  <Popover target={<Avatar src="https://randomuser.me/api/portraits/men/32.jpg">Badge with popover</Avatar>}>
    PopoverContent
  </Popover>
</AvatarGroup>
```

When I do this the avatar isn't displayed, is there a workaround or am I missing something ? (Same is happening when I try with the Tooltip component)

Thanks for your help. 🙏

**Comment by rtivital** [comment, marked as answer]
Currently, it is not supported, Popover renders an additional div element that makes it incompatible with Avatars group. We will fix that in version 5.0.

**Reply by Kayoshi-dev** [reply]
Thanks for your answer! Should I create an issue for this request or is it already planned in your roadmap?

**Reply by rtivital** [reply]
It is already planned and partially implemented

**Reply by Kayoshi-dev** [reply]
Cool! Thanks for your help, can't wait to use it!
